grpc/read_job.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "elliptics.grpc.fb.h"
+#include "job.hpp"
+#include "library/n2_protocol.hpp"
+
+namespace ioremap { namespace grpc {
+
+class read_job_t : public job_t {
+public:
+	using request_t = elliptics::n2::read_request;
+	using response_t = elliptics::n2::read_response;
+	using rpc_request_t = flatbuffers::grpc::Message<fb_grpc_dnet::ReadRequest>;
```

Comment:
Serializers use these synonyms.

Reply:
In current PR I don't see use of these synonyms outside of job. Or tell me where these use-cases are situated. If you only plan to use, it's not clear: to do the part of your plan in this commit, and the another part as a promise that is not guaranteed to be satisfied. If it will be commit, where you move `using` from private section, joining this action with using this typedef outside of the class - it will look clear. It's my opinion.

Reply:
There: https://github.com/interiorem/elliptics/pull/245/files#diff-f36b8d92c5de927bcac102d4e8a64f87R43